### PR TITLE
ref(trace): address feedback in trace view

### PR DESCRIPTION
**Pull request by JonasBa**
Addresses some feedback from @gggritso 

- add cursor pointer to rows,
- move txn warning to drawer and fix flashing
- increase drawer handle size
- persist click selection styles
- fix tab title

### CODE REVIEW

File: static/app/components/events/interfaces/spans/newTraceDetailsSpanDetails.tsx [hunk 1]
```diff
@@ -599,6 +602,31 @@ function NewTraceDetailsSpanDetail(props: SpanDetailProps) {
...
+
+    const parsedURL = safeURL(url);
+    const queryString = qs.parse(parsedURL?.search ?? '');
+
+    return parsedURL ? (
+      <Fragment>
+        <Row title={t('HTTP Method')}>{method}</Row>
+        <Row title={t('URL')}>
+          {parsedURL ? parsedURL?.origin + parsedURL?.pathname : 'failed to parse URL'}
+        </Row>
```
Comment (gggritso) [inline static/app/components/events/interfaces/spans/newTraceDetailsSpanDetails.tsx:617]: Love it, but I think this should include the query parameters so people can copy/paste the full URL

**Review by gggritso — APPROVED** [review]
Very nice!

File: static/app/components/events/interfaces/spans/newTraceDetailsSpanDetails.tsx (hunk 1 above)
Comment (JonasBa) [reply static/app/components/events/interfaces/spans/newTraceDetailsSpanDetails.tsx:617]: lets make a separate row - I think having it separate is nice bc you dont need to visually scan for the query string in the end

Comment (JonasBa) [reply static/app/components/events/interfaces/spans/newTraceDetailsSpanDetails.tsx:617]: Actually @gggritso the full URL is already displayed at the bottom!

**Pull request merged**